tests/integration-service/integration.go

```diff
@@ -92,6 +96,21 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 			It("checks if all of the integrationPipelineRuns passed", Label("slow"), func() {
 				Expect(f.AsKubeDeveloper.IntegrationController.WaitForAllIntegrationPipelinesToBeFinished(testNamespace, applicationName, snapshot)).To(Succeed())
 			})
+
+			It("checks if the status of integration test is reported in the Snapshot", func() {
+				Eventually(func() error {
+					snapshot, err = f.AsKubeAdmin.IntegrationController.GetSnapshot(snapshot.Name, "", "", testNamespace)
+					Expect(err).ShouldNot(HaveOccurred())
+
+					statusDetail, err := f.AsKubeDeveloper.IntegrationController.GetIntegrationTestStatusDetailFromSnapshot(snapshot, integrationTestScenario.Name)
+					Expect(err).ToNot(HaveOccurred())
```

Comment:
Do we want to exit the Eventually block (without retrying) when either of these 2 Expect assertions fails? If no, then we would need to replace them with:
```
if err != nil {
  return err
}
```

Reply:
if at that point there is no record about test (at least pending state) we should just burn it down